identifiers/country-jp/prefectures.csv

```diff
@@ -1,48 +1,50 @@
-ocd-division/country:jp/prefecture:jp-41,Saga
-ocd-division/country:jp/prefecture:jp-11,Saitama
-ocd-division/country:jp/prefecture:jp-25,Shiga
-ocd-division/country:jp/prefecture:jp-32,Shimane
-ocd-division/country:jp/prefecture:jp-22,Shizuoka
-ocd-division/country:jp/prefecture:jp-09,Tochigi
-ocd-division/country:jp/prefecture:jp-36,Tokushima
-ocd-division/country:jp/prefecture:jp-13,Tokyo
-ocd-division/country:jp/prefecture:jp-31,Tottori
-ocd-division/country:jp/prefecture:jp-16,Toyama
-ocd-division/country:jp/prefecture:jp-30,Wakayama
-ocd-division/country:jp/prefecture:jp-06,Yamagata
-ocd-division/country:jp/prefecture:jp-35,Yamaguchi
-ocd-division/country:jp/prefecture:jp-19,Yamanashi
\ No newline at end of file
+ocd-division/country:jp/prefecture:23,Aichi
+ocd-division/country:jp/prefecture:05,Akita
+ocd-division/country:jp/prefecture:02,Aomori
+ocd-division/country:jp/prefecture:12,Chiba
+ocd-division/country:jp/prefecture:38,Ehime
+ocd-division/country:jp/prefecture:18,Fukui
+ocd-division/country:jp/prefecture:40,Fukuoka
+ocd-division/country:jp/prefecture:07,Fukushima
+ocd-division/country:jp/prefecture:21,Gifu
+ocd-division/country:jp/prefecture:10,Gunma
+ocd-division/country:jp/prefecture:34,Hiroshima
+ocd-division/country:jp/prefecture:01,Hokkaido
+ocd-division/country:jp/prefecture:28,Hyōgo
+ocd-division/country:jp/prefecture:08,Ibaraki
+ocd-division/country:jp/prefecture:17,Ishikawa
+ocd-division/country:jp/prefecture:03,Iwate
+ocd-division/country:jp/prefecture:37,Kagawa
+ocd-division/country:jp/prefecture:46,Kagoshima
+ocd-division/country:jp/prefecture:14,Kanagawa
+ocd-division/country:jp/prefecture:39,Kōchi
+ocd-division/country:jp/prefecture:43,Kumamoto
+ocd-division/country:jp/prefecture:26,Kyoto
+ocd-division/country:jp/prefecture:24,Mie
+ocd-division/country:jp/prefecture:04,Miyagi
+ocd-division/country:jp/prefecture:45,Miyazaki
+ocd-division/country:jp/prefecture:20,Nagano
+ocd-division/country:jp/prefecture:42,Nagasaki
+ocd-division/country:jp/prefecture:29,Nara
+ocd-division/country:jp/prefecture:15,Niigata
+ocd-division/country:jp/prefecture:44,Ōita
+ocd-division/country:jp/prefecture:33,Okayama
+ocd-division/country:jp/prefecture:47,Okinawa
+ocd-division/country:jp/prefecture:27,Osaka
+ocd-division/country:jp/prefecture:41,Saga
+ocd-division/country:jp/prefecture:11,Saitama
+ocd-division/country:jp/prefecture:25,Shiga
+ocd-division/country:jp/prefecture:32,Shimane
+ocd-division/country:jp/prefecture:22,Shizuoka
+ocd-division/country:jp/prefecture:09,Tochigi
+ocd-division/country:jp/prefecture:36,Tokushima
+ocd-division/country:jp/prefecture:13,Tokyo
+ocd-division/country:jp/prefecture:31,Tottori
+ocd-division/country:jp/prefecture:16,Toyama
+ocd-division/country:jp/prefecture:30,Wakayama
+ocd-division/country:jp/prefecture:06,Yamagata
+ocd-division/country:jp/prefecture:35,Yamaguchi
+ocd-division/country:jp/prefecture:19,Yamanashi
+ocd-division/country:jp/at_large_district:tottori-shimane,Tottori-Shimane

```

Comment:
I think we should create distinct OCD IDs for all House of Councilor districts instead of just for the two which are made up of multiple prefectures, that way they can more easily be marked invalid if redistricting happens again.

It looks like common naming structure is used for all the districts, at least on English Wikipedia: https://en.wikipedia.org/wiki/Ishikawa_at-large_district

Reply:
Ack, added a new file for all the at-large districts for House of Councillors